FDRS_Gateway/fdrs_functions.h

```diff
@@ -220,11 +216,7 @@ void sendLog()
   for (int i = 0; i < ln; i++)
   {
     char linebuf[34]; // size depends on resulting length of the formatting string
-    #ifdef USE_WIFI
-    sprintf(linebuf, "%ld,%d,%d,%g\r\n", timeClient.getEpochTime(), theData[i].id, theData[i].t, theData[i].d);
-    #else
-    sprintf(linebuf, "%ld,%d,%d,%g\r\n", seconds_since_reset, theData[i].id, theData[i].t, theData[i].d);
-    #endif
+    sprintf(linebuf, "%lld,%d,%d,%g\r\n", time(nullptr), theData[i].id, theData[i].t, theData[i].d);
```

Comment:
Removed seconds since reset? I was just warming up to it... :D

Reply:
you can just call time(nullptr). This returns the same value

Reply:
no need to reinvent the wheel

Reply:
I'm jealous you guys have all this time to work on coding this project.

Reply:
Well... I used to spend about a month thinking for every day I spent coding this project. I'm very grateful to have helpful people to push the project along now because I still don't (technically) have much time for it.